poi-ooxml/src/main/java/org/apache/poi/xssf/streaming/SXSSFSheet.java

```diff
@@ -205,6 +206,10 @@ public void removeRow(Row row) {
                 return;
             }
         }
+    	// BugZilla 67646: allow reading all the content
+        if (row.getSheet() == _sh) {
+        	_sh.removeRow(row);
+        }
```

Comment:
I don't understand the intent here. row.getSheet() is checked right at the start of the method. And the row is removed in the loop. AFAIK the row is not updated when that happens, so row.getSheet() will still return `this`. So what should happen here?

Reply:
My mistake, I will remove this code